client/lib/importer/importer-config.js

```diff
@@ -180,7 +180,7 @@ function getConfig( { siteTitle = '' } = {} ) {
 		optionalUrl: {
 			title: translate( 'Substack Newsletter URL' ),
 			description: translate(
-				'An optional Substack Newsletter URL to import comments and author information.'
+				'Recommended: A Substack Newsletter URL to import comments and author information.'
 			),
 			invalidDescription: translate( 'Enter a valid Substack Newsletter URL (%(exampleUrl)s).', {
 				args: { exampleUrl: 'https://newsletter.substack.com/' },
```

Comment:
It looks like newsletter.substack.com is owned by... someone. 🙂 Is there a better URL we can use for the example?

Reply:
Changed to `example-newsletter.substack.com` that nobody seems to have squatted. Yet. 😀

Reply:
It'd probably be worth squatting it ourselves, then, and maybe putting a little bit of placeholder content there. That's not a blocker for this PR, though.